src/utils/enrichers.ts

```diff
@@ -75,3 +80,16 @@ export const fillSingleProject = async (
     projectId: data.projects[0].id,
   };
 };
+
+export const fillSingleOrg = async (
+  props: CommonProps & Partial<Pick<OrgScopeProps, 'orgId'>>,
+) => {
+  if (props.orgId) {
+    return props;
+  }
+  const { data } = await props.apiClient.getCurrentUserOrganizations();
+  if (data.organizations.length === 0) {
+    throw new Error('No organizations found');
+  }
+  return { ...props, orgId: data.organizations[0].id };
```

Comment:
so it always defaults to the first organization? is it intended for use with org API keys or something?

Reply:
This was an oversight. I was missing a conditional (like for projects above). Added.